Zend/zend_attributes.c

```diff
@@ -263,15 +263,12 @@ ZEND_API zend_internal_attribute *zend_internal_attribute_get(zend_string *lcnam
 void zend_register_attribute_ce(void)
 {
 	zend_internal_attribute *attr;
-	zend_class_entry ce;
-	zend_string *str;
-	zval tmp;
 
 	zend_hash_init(&internal_attributes, 8, NULL, free_internal_attribute, 1);
 
-	INIT_CLASS_ENTRY(ce, "Attribute", class_Attribute_methods);
-	zend_ce_attribute = zend_register_internal_class(&ce);
-	zend_ce_attribute->ce_flags |= ZEND_ACC_FINAL;
+	zend_ce_attribute = register_class_Attribute();
+	attr = zend_internal_attribute_register(zend_ce_attribute, ZEND_ATTRIBUTE_TARGET_CLASS);
```

Comment:
We could probably generate this line as well.... But it would be a bit embarrassing to mark up attributes via PHP-DOC (`@attribute`), isn't it? :D